app/views/users/index.html.erb

```diff
@@ -1,8 +1,11 @@
 <h1>Users</h1>
 <ul>
   <% @users.each do |user| %>
-    <li>id: <%= user.id %>
-    <span>name: <%= user.name %></span>
-    </li>
+    <a href="<%= user_posts_path(user) %>">
+      <div class="user-box">
+        <%= render partial: 'shared/user_photo', locals: { user: user } %>
+        <%= render partial: 'shared/user_details', locals: { user: user } %>
+      </div>
+    </a>
```

Comment:
- [ ] User page should render the user's 3 most recent post but currently it is showing all posts please address that